knapSack.py

```diff
@@ -0,0 +1,34 @@
+# Knapsack Problem
+
+# Time Complexity :  O(n) where n is the size of the dp matrix 
```

Comment:
Time Complexity: O(maxWeight * number of weights)
Space Complexity: O(maxWeight * number of weights)